### PR TITLE
Show `<DefaultMarker>` when children are only `<MapView.Callout>`

### PR DESCRIPTION
**Pull request by progfay**
```jsx
<MapView>
	<MapView.Marker>
		<MapView.Callout />
	<MapView.Marker>
</MapView>
```

[Image from Gyazo](https://gyazo.com/8ffab0fd06c2c5ccf8893e29072c568c)

### CODE REVIEW

File: src/Marker.js [hunk 1]
```diff
@@ -44,7 +45,12 @@ class MapMarker extends Component {
   }
 
   render() {
-    if (!this.props.children) return <DefaultMarker {...this.props} />;
+    let hasOnlyCalloutChildren = true;
+
+    React.Children.forEach(this.props.children, child => {
```
Comment (kasajei) [inline src/Marker.js]: I think you should use `reduce` like below.
```
let hasOnlyCalloutChildren = this.props.children.reduce(child =>{}, true)
```

Comment (progfay) [reply src/Marker.js]: On `React.Children` , `.reduce` is not implemented...

https://ja.reactjs.org/docs/react-api.html#reactchildren

Comment (progfay) [reply src/Marker.js]: Use `React.Children.toArray.reduce`

```js
const hasOnlyCalloutChildren = React.Children.toArray(this.props.children).reduce(
  (acc, child) => {},
  true
);
```

**Comment by kasajei** [comment]
LGTM

**Pull request merged**